network-errors/src/commonMain/kotlin/dev/icerock/moko/network/errors/NetworkExceptionMappers.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.network.errors
+
+import dev.icerock.moko.errors.mappers.ExceptionMappersStorage
+import dev.icerock.moko.network.exceptions.ErrorException
+import dev.icerock.moko.network.exceptions.ValidationException
+import dev.icerock.moko.network.isNetworkConnectionError
+import dev.icerock.moko.resources.StringResource
+import dev.icerock.moko.resources.desc.CompositionStringDesc
+import dev.icerock.moko.resources.desc.ResourceFormatted
+import dev.icerock.moko.resources.desc.StringDesc
+import dev.icerock.moko.resources.desc.desc
+import kotlinx.serialization.SerializationException
+
+/**
+ * Registers all default exception mappers of the network module to [ExceptionMappersStorage].
+ */
+fun ExceptionMappersStorage.registerAllNetworkMappers(
+    networkConnectionErrorText: StringResource = MR.strings.networkConnectionErrorText,
+    unauthorizedErrorText: StringResource = MR.strings.unauthorizedErrorText,
+    notFoundErrorText: StringResource = MR.strings.notFoundErrorText,
+    accessDeniedErrorText: StringResource = MR.strings.accessDeniedErrorText,
+    internalServerErrorText: StringResource = MR.strings.internalServerErrorText
+): ExceptionMappersStorage {
+    return condition<StringDesc>(
+        condition = { it.isNetworkConnectionError() },
+        mapper = { networkConnectionErrorText.desc() }
+    ).condition<StringDesc>(
+        condition = { it is SerializationException },
+        mapper = { MR.strings.serializationErrorText.desc() }
```

Comment:
should be passed from argument. or maybe change to dsl?

Reply:
Fixed